drgn_tools/block.py

```diff
@@ -216,21 +217,21 @@ def for_each_mq_pending_request(q: Object) -> Iterable[Tuple[Object, Object]]:
                 yield (hwq, rq)
 
 
-def rq_pending_time_ms(rq: Object) -> int:
+def rq_pending_time_ns(rq: Object) -> int:
     """
-    Get io pending time in ms
+    Get io pending time in ns
 
     :param rq: ``struct request *`` or ``struct request``
     :returns: request pending time
     """
     prog = rq.prog_
     if has_member(rq, "start_time"):
         # TODO: arch specific
-        return prog["jiffies_64"] - rq.start_time
+        return (prog["jiffies_64"] - rq.start_time).value_() * 1000000
```

Comment:
Since drgn v0.0.23, "jiffies" is a special-cased variable that Drgn maintains for us. Since you touched this code, please go ahead and remove the todo and change it to `prog["jiffies"]`.

Reply:
done